typings/types/index.d.ts

```diff
@@ -13,7 +13,19 @@ declare module 'types' {
 
     export type Errors = Object[]
 
-    export interface Config {
+    export interface ProfileConfig {
```

Comment:
Is this still required, now that we're using the TS code?

Reply:
right no